manifests/database/postgresql.pp

```diff
@@ -68,10 +68,16 @@
     }
   }
 
+  if $database_port != undef {
+    $db_port = $database_port
+  } else {
+    $db_port = '5432'
+  }
```

Comment:
Maybe more readable as:

```suggestion
  $db_port = pick($database_port, '5432')
```

([from stdlib](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/REFERENCE.md#pick)) or ([using built-in functions](https://puppet.com/docs/puppet/7/function.html#lest)):

```suggestion
  $db_port = $database_port.lest || { '5432' }
```

Reply:
True, I was simply trying to keep the same syntax from line 48 that is doing the same thing:

```
48  if $database_port != undef {
49    $port = "-p ${database_port} "
50  } else {
51    $port = ''
52  }
```

But i’ll gladly make my part more readable :+1:

Reply:
Ah yes, we sometimes lack this context when reviewing :-) When facing this, I generally work on my feature in a first commit, and improve / enhance / fix style in a separate commit in the same PR :-D

Reply:
No problem.  Would you like me to fix line 48 like so:
```
$port = $database_port ? {
  undef   => '',
  default => "-p $database_port",
 }
```
I would slightly consider it cleaner code, would you?